Rok 80 werbung zu sdk hinzufugen

kaevinio opened the pull request:
Hier ist ROK-82 gemerged, also den am besten zuerst abnehmen

kaevinio committed:
ROK-80: Add RoadKit banner
- Created banner to be displayed in topic and changelog list

annanna commented on Sources/RoadKitSwift/Views/Components/LogoView.swift:
```diff
@@ -0,0 +1,37 @@
+    var body: some View {
+        HStack(spacing: LayoutValues.minorPadding) {
+            VStack(spacing: LayoutValues.minorPadding / 6) {
+                Text("Provided by")
```
soll das auch mit in die Localizables?

annanna commented on Sources/RoadKitSwift/Views/Components/LogoView.swift:
```diff
@@ -0,0 +1,37 @@
+                    .font(.callout)
+                    .frame(maxWidth: .infinity, alignment: .leading)
+                
+                Text("RoadKit")
```
und das hier

annanna commented on Sources/RoadKitSwift/Views/List/RoadmapView.swift:
```diff
@@ -25,6 +25,9 @@ public struct RoadmapView: View {
                         navigationBar.navigationItem(image: Image(systemName: "chevron.down"), color: foregroundColor, action: dismissView)
                     }
                 
+                ZStack(alignment: .bottom) {
```
das liegt grad hinter der Liste wenn man noch scrollen muss
<img width="865" alt="image" src="https://user-images.githubusercontent.com/4865576/234864011-c529b588-fea6-4253-8274-ada922c5fcdd.png">

annanna commented on Sources/RoadKitSwift/Views/Components/LogoView.swift:
```diff
@@ -0,0 +1,37 @@
+
+import SwiftUI
+
+struct LogoView: View {
```
im Ticket steht noch, dass es nen Link auf die Website geben muss (oder dass die View hier nen Button ist würd ich mal sagen)

kaevinio committed:
- Moved non-localized strings to Strings-file
- Show provided by view to bottom of list instead of background
- Set link to RoadKit website for tapping on provided by view

kaevinio committed:
- FIX: Background color for provided by view
- FIX: Bottom inset for roadmap list
